docs: Explain access logs filtering (#1315)
kyma-project/istio: docs/user/tutorials/01-45-enable-istio-access-logs.md

    ```bash
    kubectl -n istio-system get telemetries.telemetry.istio.io
    ```
<!-- tabs:end -->

### Filter Access Logs

> Often access logs emmited by Envoy does not contain data relevant for your observations, especially when the traffic is not based on a HTTP based protocol. In such situation you can directly configure the Envoys of Istio to filter out logs using a filter expression. For that you can leverage the same [Istio Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#AccessLogging) as you used to enable the access logs to also filter them. Here, you can define a filter expression leveraging the typical [Envoy attributes](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes) to formulate which logs to **keep**.

Reviewer: ```suggestion
Often, access logs emitted by Envoy do not contain data relevant to your observations, especially when the traffic is not based on an HTTP-based protocol. In such a situation, you can directly configure the Istio Envoys to filter out logs using a filter expression. To filter access logs, you can leverage the same [Istio Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#AccessLogging) that you used to enable them. To formulate which logs to **keep**, define a filter expression leveraging the typical [Envoy attributes](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes).
```
